### ISSUE TITLE
Comment location issue

### ISSUE THREAD

**Issue by vjeux** [postcss/postcss-scss#63]
```css
@font-face  {
  src: url(if(
    $bootstrap-sass-asset-helper,
    twbs-font-path('#{$icon-font-path}#{$icon-font-name}.eot'),
    '#{$icon-font-path}#{$icon-font-name}.eot'
  ));
}

// Catchall baseclass
.glyphicon {
}
```

Outputs for the comment

```js
    {
      "raws": {
        "before": "\n\n",
        "inline": true,
        "left": " ",
        "right": "",
        "text": "Catchall baseclass"
      },
      "type": "css-comment",
      "source": {
        "start": {
          "line": 5,
          "column": 1
        },
        "input": {
          "css": "@font-face  {\n  src: url(if(\n    $bootstrap-sass-asset-helper,\n    twbs-font-path('#{$icon-font-path}#{$icon-font-name}.eot'),\n    '#{$icon-font-path}#{$icon-font-name}.eot'\n  ));\n}\n\n// Catchall baseclass\n.glyphicon {\n}\n",
          "id": "<input css 1>"
        },
        "end": {
          "line": 5,
          "column": 21
        }
      },
      "text": "Catchall baseclass"
    },
```

It says line 5, column 1-21. But this is `'#{$icon-font-pa` and not `// Catchall baseclass`.

**Comment by vjeux** [comment]
This parses correctly for postcss-less fwiw.

**Comment by ai** [comment]
Thanks for report. I look into it, when I will return to good Internet at 9 June.

**Comment by vjeux** [comment]
Enjoy your vacation! I have a workaround in place so it's not super urgent.

**Comment by ai** [comment]
I am back! :) Very strange, that you got `css-comment` type, because PostCSS or `postcss-scss` doesn’t use it. Can you show your JS code?

**Comment by ai** [comment]
But I found same `Comment#source` problem in `postcss-scss`. Main problem is because PostCSS was created with thoughts that `url()` could not have `\n` (it is very special token, not just function).

**Comment by ai** [comment]
Do I understand correctly that `url()` could not contains `\n` in normal CSS?
https://www.w3.org/TR/css-syntax-3/#url-token-diagram

In this case we add this hack only for `postcss-scss`.

**Comment by vjeux** [comment]
I have to say, I have no idea what is happening. I don't understand the CSS spec nor how postcss works to be of help :(

**Comment by ai** [comment]
OK, tomorrow I will add fix only to `postcss-scss` :)

**Comment by vjeux** [comment]
postcss-less works fine, so if you fix it here, it'll work well for prettier :) thanks!

**Comment by ai** [comment]
Done 81f8130

**Comment by ai** [comment]
I will release it when we get decision for https://github.com/postcss/postcss-scss/issues/64

**Comment by vjeux** [comment]
Thanks!

**Comment by ai** [comment]
Released in 1.0.1
